otherlibs/systhreads/st_stubs.c

```diff
@@ -377,8 +377,9 @@ static void caml_thread_reinitialize(void)
      the effective owner of the lock. So there is no need to run
      st_masterlock_acquire (busy = 1) */
   st_masterlock *m = Thread_lock(Caml_state->id);
-  m->init = 0; /* force initialization */
-  st_masterlock_init(m);
+  m->init = 0; /* force reinitialization */
+  if (0 != st_masterlock_init(m))
```

Comment:
Is `0 != st_masterlock_init(m)` better than `st_masterlock_init(m) != 0`?

Reply:
The house style in the runtime is mostly the latter, with only a few sprinklings of the `(0 == ...)` style that I can find in runtime/sys.c

Reply:
Thanks, fixed. Comments about style are welcome, to help fix the inconsistency of the code base.